lib/gollum-lib/markups.rb

```diff
@@ -28,11 +28,24 @@ def executable_exists?(name)
 
 include Gollum::MarkupRegisterUtils
 
+module GitHub
+  module Markup
+    class Markdown < Implementation
+      def implementation_name
+        @implementation_name ||= MARKDOWN_GEMS.keys.detect {|gem_name| try_require(gem_name) }
+      end
+    end
+  end
+end
+
 module Gollum
   class Markup
     GitHub::Markup::Markdown::MARKDOWN_GEMS['kramdown'] = proc { |content|
         Kramdown::Document.new(content, :auto_ids => false, :smart_quotes => ["'", "'", '"', '"'].map{|char| char.codepoints.first}).to_html
     }
+    GitHub::Markup::Markdown::MARKDOWN_GEMS['pandoc-ruby'] = proc { |content|
+        PandocRuby.convert(content, :s, :from => :markdown, :to => :html, :filter => 'pandoc-citeproc')
```

Comment:
Possibly make this conditional on the availability of the pandoc-ruby gem? If it isn't available it shouldn't matter to github-markup though.